OlxProject/Tools/driver.rb

```diff
@@ -1,13 +1,16 @@
 require 'rubygems'
 require 'selenium-webdriver'
 
-public def StartTest
+public def start_test
```

Comment:
why use explicit declaring of public? what is the default method scope when creating any method?

Reply:
I edited the code, deleted 'public' and forgot to do this here.